.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: aPXmpfA8lSNBRSDziteIAfJWNnuhQXVHv
```

Comment:
Please load the repo token from the `env` file 

you can say repo_token=`process.env.REPO_TOKEN`
then in `.env` file you do `REPO_TOKEN` =`aPXmpfA8lSNBRSDziteIAfJWNnuhQXVHv`